_publications/2022-07-26-portmaster-vs-glasswire.md

```diff
@@ -0,0 +1,208 @@
+---
+# possible values: blog video podcast
+category: blog
+layout:   blog
+title: "What Makes Portmaster A Great Alternative to GlassWire"
+date: 2022-07-26
+author: safing
+custom_thumbnail_name: portmaster-vs-glasswire
+portmaster_plug: true
+summary: Both Portmaster and GlassWire are powerful applications, are well designed and are easy to set up. Portmaster has excellent privacy defaults, which protect you from mass surveillance across the board. With GlassWire you have to build your own defenses from scratch. Another weak point of GlassWire is that it is proprietary. Portmaster on the other hand is free and open source, which is why we believe most people will trust Portmaster more.
+---
+
+## Overview Chart
+
+<div class="card-table">
+  <table>
+    <colgroup class="card-table-colgroup">
+      <col class="card-table-colgroup-col">
+      <col class="card-table-colgroup-col-highlight">
+      <col class="card-table-colgroup-col">
+    </colgroup>
+    <thead class="card-table-header">
+      <tr class="card-table-header-row">
+        <th class="card-table-header-row-cell"></th>
+        <th class="card-table-header-row-cell">Portmaster</th>
+        <th class="card-table-header-row-cell">GlassWire</th>
+      </tr>
+    </thead>
+    <tbody class="card-table-body">
+      <tr class="card-table-body-row">
+        <td class="card-table-body-row-cell-title">Open Source</td>
+        <td class="card-table-body-row-cell">
+          <a class="card-table-body-row-cell-link" href="{{ site.github_url }}/portmaster">
+            <i class="icon-tick text-2xl text-green-500"></i>
+          </a>
+        </td>
+        <td class="card-table-body-row-cell">
+          <i class="icon-cross text-2xl text-red-500"></i>
+        </td>
+      </tr>
+      <tr class="card-table-body-row">
+        <td class="card-table-body-row-cell-title">Setup</td>
+        <td class="card-table-body-row-cell"><strong>Easy</strong></td>
+        <td class="card-table-body-row-cell"><strong>Easy</strong></td>
+      </tr>
+      <tr class="card-table-body-row">
+        <td class="card-table-body-row-cell-title">Protection Type</td>
+        <td class="card-table-body-row-cell"><strong>Device</strong></td>
+        <td class="card-table-body-row-cell"><strong>Device</strong></td>
+      </tr>
+      <tr class="card-table-body-row">
+        <td class="card-table-body-row-cell-title">System Integration</td>
+        <td class="card-table-body-row-cell"><strong>Kernel Extension</strong></td>
+        <td class="card-table-body-row-cell"><strong>Windows Filtering Platform</strong></td>
```

Comment:
We also use WFP, but in the kernel, if you feel that is important.

Reply:
I feel that is elaborated in the paragraphs